website/content/docs/job-specification/volume_mount.mdx

```diff
@@ -48,9 +49,15 @@ updates to remove a volume that it depends on.
   specify that it is `read_only` on a per mount level using the `read_only`
   option here.
 
+- `propagation_mode` `(string: "")` - Configure [docker bind propagation][]. Can
+  be one of `private`、`host-to-task` or `bidirectional`. Empty string and
+  `private` maps to `rprivate`, `host-to-task` maps to `rslave`, and
+  `bidirectional` maps to `rshared`.
+
 For examples of how to use [HCL2] interpolation for fine-grained control of
 volumes, see [Volume Interpolation].
 
 [volume]: /docs/job-specification/volume 'Nomad volume Job Specification'
 [volume interpolation]: /docs/job-specification/volume#volume-interpolation
 [hcl2]: /docs/job-specification/hcl2
+[docker bind propagation]: https://docs.docker.com/storage/bind-mounts/#configure-bind-propagation
```

Comment:
```suggestion
[volume]: /nomad/docs/job-specification/volume 'Nomad volume Job Specification'
[volume interpolation]: /nomad/docs/job-specification/volume#volume-interpolation
[hcl2]: /nomad/docs/job-specification/hcl2
[docker bind propagation]: https://docs.docker.com/storage/bind-mounts/#configure-bind-propagation
```
Hello 👋🏻 We've[ recently migrated](https://github.com/hashicorp/nomad/pull/15779) all links to point to their full relative path from developer.hashicorp.com. To resolve your conflict with `main`, please prefix these paths with the product slug: `nomad`.